packages/bot-web-ui/src/pages/dashboard/dashboard.scss

```diff
@@ -59,161 +45,51 @@
-                            }
-                        }
-                    }
-                }
-            }
-
-            &__content {
-                @include mobile {
-                    width: 100%;
-                    overflow-x: auto;
-
-                    &--no-result {
-                        @include align-center;
-                        flex-direction: column;
-                        justify-content: flex-start;
-                        height: 100vh;
-                        margin-top: 5rem;
-
-                        svg {
-                            margin-bottom: 2.4rem;
-                        }
-                        h1 {
-                            margin-bottom: 0.8rem;
-                        }
-                        span {
-                            word-break: break-word;
-                            text-align: center;
-                        }
-                    }
-                }
+.db {
+    --icon-height: 2.4rem;
+    --icon-width: 2.4rem;
+    --border-radius: 2.4rem;
 
-                &--dashboard {
-                    &__tabs {
-                        display: flex;
-                        justify-content: space-between;
-
-                        @include mobile {
-                            flex-direction: column;
-                            background-color: var(--general-section-1);
-                        }
-                    }
-                }
-            }
+    &-sidebar {
+        position: relative;
+        background-color: var(--general-main-1);
+        padding: 2.4rem;
+        height: 100%;
 
-            &__item {
-                border-top-left-radius: 1rem;
-                border-top-right-radius: 1rem;
-                height: 4.8rem;
-                @include align-center;
-
-                &__loaded {
-                    text-align: left;
-                    padding-left: 1.5rem;
-                }
-
-                svg {
-                    width: 1.6rem;
-                    height: 1.6rem;
-                    padding-right: 0;
-                    margin-right: 0.8rem;
-                }
-            }
+        &--block {
+            display: block;
         }
-    }
-}
 
-.w-100 {
-    width: 100%;
-}
-
-.nm-48 {
-    margin-top: -4.8rem;
-}
-
-.db-sidebar {
-    position: relative;
-    background-color: var(--general-main-1);
-    padding: 2.4rem;
-    height: 100%;
+        display: none;
 
-    &--block {
-        display: block;
-    }
-
-    display: none;
-
-    &__images {
-        background: var(--general-section-6);
+        &__images {
+            background: var(--general-section-6);
+        }
     }
-}
-
-.db-info-panel {
-    &__close-action {
-        position: absolute;
-        right: 1.4rem;
-        top: 1.4rem;
-        height: 2rem;
-        width: 2rem;
-        line-height: 2.3rem;
-        text-align: center;
-        border-radius: 1rem;
+    &-info-panel {
+        &__close-action {
```

Comment:
as per BEM` __close-action` is not an element, it is a modifier and should be`--close-action `

Reply:
@maryia-matskevich-deriv __close-action if an Element and it is a wrapper for the close icon.
Can you please check again and let me know if there are any confusions?

Reply:
@sandeep-deriv In this case it's using the name as a noun rather than a verb, then it's an element and it takes place... Probably we can improve a name to keep it more clear, for example `__close-action-btn` depending on what it really is or whatever is there

Reply:
@sandeep-deriv It's certainly a matter of taste, but the name should match the element/block as a noun, just like you call a modifier a verb because that's the custom. This helps maintain consistency and clarity in naming elements within a project. 
But you may do what you think is best😌